homeassistant/components/wemo/number.py

```diff
@@ -0,0 +1,70 @@
+"""Number platform support for WeMo."""
+from __future__ import annotations
+
+import asyncio
+import math
+
+from pywemo import Dimmer
+
+from homeassistant.components.number import NumberEntity, NumberMode
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN as WEMO_DOMAIN
```

Comment:
This isn't strictly necessary.

Reply:
Just to clarify, it's the `as WEMO_DOMAIN` that isn't necessary?
Changed in [e3de035](https://github.com/home-assistant/core/pull/78217/commits/e3de0358b52500de49001d62c1fc64310df42b0c)

Reply:
Correct. That would only be required if you were importing a different `DOMAIN` constant from elsewhere.